app/controllers/conference_attendances_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ConferenceAttendancesController < ApplicationController
+
+  def update
+    @conference_attendance = ConferenceAttendance.find(attendance_params[:id])
+    @team = Team.find(attendance_params[:team_id])
+
+    if @conference_attendance.update_attributes(attendance_params)
+      redirect_to @team, notice: 'Your attendance was successfully recorded.'
+    else
+      render json: @team.errors, status: :unprocessable_entity
```

Comment:
I presume that using `@team.errors` is probably inteded to be `@conference_attendance`? `@team` is not saved/validated in this action so it can not have any errors.